transcendence-app/src/game/game.queue.service.ts

```diff
@@ -29,13 +28,13 @@ export class GameQueueService {
   private gameQueue: QueuedUser | null = null;
 
   constructor(
-    private gamesOngoing: IGamesOngoingRepository,
```

Comment:
didn't we say that this could be readonly?

Reply:
We moved the ongoing games to the game service, and now the queue service uses that one to check if a player is in a game.

The ongoing games are read-only for the queue service meaning that it only reads the game status (get) and doesn't write (create, update, delete)